tools/qontract_cli.py

```diff
@@ -2577,6 +2578,27 @@ def osd_component_versions(ctx):
     print_output(ctx.obj["options"], data, columns)
 
 
+@get.command()
+@click.pass_context
+def maintenances(ctx):
+    now = datetime.now(timezone.utc)
+    maintenances = maintenances_gql.query(gql.get_api().query).maintenances or []
+    maintenances = [
+        m for m in maintenances if datetime.fromisoformat(m.scheduled_end) > now
+    ]
+    data = [vars(m) for m in maintenances]
+    for m in data:
+        m["services"] = ", ".join([a.name for a in m["affected_services"]])
+    print(data)
```

Comment:
Is this for debug print or we want to pretty print `data`?

Reply:
debugging leftovers. https://github.com/app-sre/qontract-reconcile/pull/4348/commits/4178ba8c79eec8cc0e569b7d1d9b53cc73b91a04